src/mnt/mount_options.rs

```diff
@@ -22,6 +22,8 @@ pub enum MountOption {
     /// Allow the root user to access this filesystem, in addition to the user who mounted it
     AllowRoot,
     /// Automatically unmount when the mounting process exits
+    ///
+    /// `AutoUnmount` requires `AllowOther` or `AllowRoot`. If `AutoUnmount` is set and neither `Allow...` is set, then `AllowOther` will be automatically added to the options. If the FUSE configuration doesn't permit `AllowOther`, this will cause an `EFAULT` error and mounting will fail.
```

Comment:
Actually, it doesn't automatically add `AllowOther` exactly. It does set `allow_other`, but it enforces permission checks in the library so that other users can't access the mount. Can you change it to something like:
```
If `AutoUnmount` is set and neither `Allow...` is set, the FUSE configuration must permit `allow_other`, otherwise mounting will fail
```

Reply:
Fixed! Sorry you're effectively the one writing the PR now... 🤭